config-template.json

```diff
@@ -14,27 +14,26 @@
         {
           "id": "98FF26D0-2E01-11E5-9184-FEFF819CDC9F",
           "name": "s3",
-          "description": "Amazon Simple Storage Service (S3)",
+          "description": "Amazon Simple Storage Service (Amazon S3), provides developers and IT teams with secure, durable, highly-scalable object storage.",
```

Comment:
This description seems like it might be too long. 
I suggest `Amazon S3 provides developers with secure, durable, highly-scalable object storage.`